bluzelle/assetlist.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "bluzelle",
+  "assets": [
+    {
+      "description": "The native token of Bluzelle",
+      "denom_units": [
+        {
+          "denom": "ubnt",
+          "exponent": 0
+        },
+        {
+          "denom": "bnt",
+          "exponent": 6
+        }
+      ],
+      "base": "ubnt",
+      "name": "Bluzelle",
+      "display": "blz",
```

Comment:
the 'display' denom must be listed under denom_units, but 'blz' is not listed there. Is the 'bnt' and 'ubnt' a typo?

Reply:
checked out the explorer. Looks like ubnt is correct. In which case I suggest changing the 'display' denom to 'bnt'. But also adding 'blz' as an alias unit of 'bnt'.

Reply:
It is not a typo. BLZ is what we have been using as a symbol, but I understand now. I have corrected it and will push an update shortly along with the other fixes.

Reply:
yes, with the denom_unit, e.g., 
```
{
  "denom": "bnt",
  "exponent": 6,
  "aliases": [
    "blz"
  ]
}
```